src/index.html

```diff
@@ -20,8 +21,709 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple.html"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung.html"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones.html"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-and-computers.html"
```

Comment:
The `href` attribute is using a fragment identifier (`#laptops-and-computers.html`). If these are meant to link to separate HTML files, consider removing the `#` to ensure proper navigation.